core/src/main/java/org/frankframework/jdbc/datasource/TransactionalDbmsSupportAwareDataSourceProxy.java

```diff
@@ -44,8 +44,8 @@ public TransactionalDbmsSupportAwareDataSourceProxy(DataSource delegate) {
 
 	public Map<String, String> getMetaData() throws SQLException {
 		if (metadata == null) {
-			log.debug("populating metadata from getMetaData");
-			try (Connection connection = getConnection()) {
+			try (Connection connection = super.getConnection()) {
+				log.trace("populating metadata from getMetaData");
```

Comment:
Reduce spam, this ain't actually that interesting and is solely here for debugging purposes...